service-discovery/README.adoc

```diff
@@ -31,304 +30,371 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 ====
 
-// Purpose statement for the assembly
 [role="_abstract"]
-As a developer of applications and services, you can connect Quarkus applications to Kafka instances in {product-long}. https://quarkus.io/[Quarkus^] is a Kubernetes-native Java framework made for Java virtual machines (JVMs) and native compilation, and optimized for serverless, cloud, and Kubernetes environments. Quarkus is designed to work with popular Java standards, frameworks, and libraries like Eclipse MicroProfile and Spring, as well as Apache Kafka, RESTEasy (JAX-RS), Hibernate ORM (JPA), Infinispan, Camel, and many more.
+As a developer of applications and services, there might be cases where you need to connect applications deployed on a Kubernetes platform such as Red Hat OpenShift to Kafka instances created in {product-long}.
 
-OpenShift is an Enteprise Kuberentes Platform. When deploying Quarkus and any other type of application, developers can use
-the https://developers.redhat.com/blog/2019/12/19/introducing-the-service-binding-operator/[Service Binding Operator] to
-enable your applications to discover service configuration automatically in your OpenShift or Kubernetes instance.
+For example, suppose that you have two applications deployed to an OpenShift cluster; one that publishes price updates for a variety of stocks, and another that consumes the price updates for publication on a web page. You also have a Kafka instance in {product-long}. Each time there is a price update, you want to use the Kafka instance to forward the update as an event to the consuming application. To achieve this, you need a way to connect the applications in your OpenShift cluster to your Kafka instance in {product-long}. Connecting a Kubernetes application to a backing service such as {product-long} is called __service binding__.
 
-You can review all possible commands in https://github.com/redhat-developer/app-services-cli/blob/main/docs/commands/rhoas_cluster.adoc[cli documentation].
+This guide describes how to use the OpenShift Application Services (RHOAS) Operator to perform service binding for {product-long}. The Kubernetes platform used throughout this guide is Red Hat OpenShift.
 
+[id="con-about-service-binding-using-rhoas-operator_{context}"]
+== About service binding using the RHOAS Operator
 
-In this Quick Start, you'll connect your OpenShift Streams for Apache Kafka cloud service to your OpenShift cluster using the Red Hat OpenShift Application Services. (RHOAS) Operator and CLI. For more information on Red Hat OpenShift Application Services, please visit link:https://cloud.redhat.com/application-services[cloud.redhat.com.]
+Connecting a Kubernetes application to a backing service such as {product-long} is called __service binding__. To perform service binding for {product-long}, you use the OpenShift Application Services (RHOAS) Operator.
 
-.Prerequisites
```

Comment:
So this will remove the required section for access.redhat.com documentation.
Do we want this?

Reply:
Discussed in person with @wtrocki. This change was appropriate as part of refactoring draft QS content as a more long-form tutorial for inclusion in a service binding guide.